<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyQt5 Designer Beispiel GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Lass dich begrüssen</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Wie ist dein Name?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_widget"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="greet_button">
      <property name="text">
       <string>Begrüsse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="greeting_output">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>349</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
